plugins/microchip/switchtec-nvme-device.c

```diff
@@ -120,7 +122,7 @@ int pax_nvme_submit_admin_passthru_4k_rsp(int fd,
 	struct pax_nvme_device *pax;
 	struct switchtec_admin_passthru_nvme_req req;
 	struct switchtec_admin_passthru_nvme_rsp rsp;
-	size_t data_len;
+	size_t data_len = 0;
```

Comment:
Ditto

Reply:
This one is required since `data_len` is used later at line 147 and 152. If not initialized it could lead to errors later.